src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -23,20 +24,23 @@ use crate::config::{Deserialize, Deserializers};
 #[derive(Copy, Clone, Eq, PartialEq, Hash, Debug, Default, serde::Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct TimeTriggerConfig {
-    interval: TimeTriggerInterval,
+    /// Trigger interval
+    pub interval: TimeTriggerInterval,
     #[serde(default)]
-    modulate: bool,
+    /// TODO:
+    pub modulate: bool,
     #[serde(default)]
-    max_random_delay: u64,
+    /// TODO:
+    pub max_random_delay: u64,
 }
 
 #[cfg(not(feature = "config_parsing"))]
 /// Configuration for the time trigger.
 #[derive(Copy, Clone, Eq, PartialEq, Hash, Debug, Default)]
 pub struct TimeTriggerConfig {
-    interval: TimeTriggerInterval,
-    modulate: bool,
-    max_random_delay: u64,
+    pub interval: TimeTriggerInterval,
```

Comment:
Missing a TimeTriggerConfig  constructor. How was this done before? How did they construct TimeTrigger without config construction?